### PR TITLE
Adds cool existing suit to ClothesMate

### PR DESCRIPTION
**Pull request by AseFantome**
## About The Pull Request
Right now this suit spawns only in MetaStation maints, one copy, determined place. There is no other ways to acquire it.
This PR adds a "Vice Officer Suit" to ClothesMate. Now everyone can enjoy it. Premium to preserve rarity of costume.
Анимация
## Why It's Good For The Game
More drip for the drip Lord. Also it doesn't adds anything "new", just a use for almost unused existing item in code.
## Changelog
Added "Vice Officer Suit" to ClothesMate, premium category, 100 cr.

:cl: NeinAnimas
add: There is new (old not used anywhere) suit in ClothesMate. Check premium category.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
Don't add PR's that make rare things less rare without gameplay justification

**Comment by KittyNoodle** [comment]
The vice officer’s suit has been made less rare, game balance in tatters

in all seriousness the suit has baseline values for everything, literally could be assigned to all roundstart assistants and it wouldn’t change a thing

**Comment by AseFantome** [comment]
> Don't add PR's that make rare things less rare without gameplay justification

Don't add comments that makes harmless PR closed without proper justification.
You saw "exists in one copy" and auto-responded, basically.

**Comment by Helg2** [comment]
> Don't add PR's that make rare things less rare without gameplay justification

Is there any justification why it should be this rare or I don't get it?

**Comment by optimumtact** [comment]
yeah you're going to do well here.

**Comment by paganiy** [comment]
It's just a cool looking suit, why not?

**Comment by Namwty** [comment]
"Rare item"

Item spawns in same exact spot on exact same map, along with other stuff, like one specific security jumpsuit and stuff like sunglasses in some-type of box. Virtually nothing interesting/unique or anything to the suit that would justify it's "rarity".

Surely this suit wouldn't be used in any gimmicks relating to trades, but we hate those anyhow.

**Comment by itseasytosee** [comment]
based

**Comment by AseFantome** [comment]
I do not know how to prove the gameplay necessity of a cosmetic item to a person who does not respond to PMs and ignores questions on github. **Maybe I should collect a list of signatures of people who would buy this suit?** 
About "sassing". If common mimicry and indignation is too much to bear, I'm really sorry. Maybe I should think carefully before speaking in ethernet. Maybe even twice. 
The thing that triggered my fast response (that you find harsh) was obvious "I don't want to really dive into thing I'm judging about, let me respond something quickly; let me  judge only by person behavior and my feelings instead of matter of subject".
I always say what I think and say it directly. I don't care who I'm talking to. Being head coder (or whoever) doesn't gives right to act like this. I can easily show same "sassing" behavior from him, but **who cares?** 
All I want is just add this fancy unused suit (aka useless bites of code and sprites rn) to rotation for me, my friends (who asked for it) and everyone who wants it too. I don't need all this drama and showdown (aka thing that pushes new people away from TG coding). 

**TL:DR** I didn't want to hurt anyone's feelings, I just wanted this costume to appear in the game. If anyone can help me with this, then I am open to talk and ready to work.**

**Comment by AseFantome** [comment]
By the way, **"yeah you're going to do well here"** sounds like a threat. I wonder how I managed to really upset someone with this one line harmless webedit. Check other PRs with this TG for fun sake.
image

**Comment by Timberpoes** [comment]
You don't need to prove the gameplay necessity. You just need to come up with a valid gameplay justification someone from the maintainer team vibes with. 

My advice would be find the PR that introduced the items in question. Then read and understand the purpose behind said PR - what was the author trying to add to the game or accomplish.

Knowing that information, you're equipped to explain why the original item was (or your best interpretation of why it was likely to be) so rare/scarce.

That's the minimum starting point for changing how rare/scarce an item is, to understand why it's rare/scarce in the first place. Because you can use that to clearly explain why the status quo is bad with reference to why the item was added, and why your changes are good.

And try to avoid using justifications that can be trivially negated. "Rarity bad" can just be responded with "rarity good". "Because I just want to do this change, I dunno" can just be responded with "well I don't want this change".

And changes that take moderately low effort to implement (like adding an item to X vendor or changing a couple of value for the damage of a weapon) often require a disproportionate justification, because we want to discourage contributors from opening low-effort PRs tweaking numbers on whims or for utterly mundane and trivial reasons or goals.

"Rarity bad because the original PR tried to do X and Y, but this has pretty much resulted in Z outcome. This seeks to change Z outcome and then improves [some aspect of the game]" is a much more complete justification.

**Comment by dessysalta** [comment]
> And changes that take moderately low effort to implement (like adding an item to X vendor or changing a couple of value for the damage of a weapon) often require a disproportionate justification, because we want to discourage contributors from opening low-effort PRs tweaking numbers on whims or for utterly mundane and trivial reasons or goals.

Thank you for giving a better response than one of the main devs without a kneejerk and snide comment to a legitimate criticism and concern. I don't really know what else to say other than I felt like I had to jump in here, because it legitimately frustrates me to see a headcoder attempt to alienate as many new contributors as possible.

**Comment by AseFantome** [comment]
Thank you, kind man, for a good answer.
Lets follow your advice step by step.

### 1.  Understand why item is in current state.
_/obj/item/clothing/under/misc/vice_officer_ exists at least since 2012 c86c8e15054b935c8885be088fad3febcae82f78 (under old _/obj/item/clothing/under/rank/vice_ path). Apparently, even earlier, but this is too much of a dig into history. 
It lies in MetaStation maints since first commit to MetaStation bc95cc8687aa3dc062bf24cf9d7619eefaa6ba12 alongside with other stuff.
Item had no further history - it was just unused.
Suit has no unique stats - **BIOHAZARAD I**, **WOUNDING I**. It's unique only in terms of sprite.
**I draw a conclusion that item was inherited by the build, randomly added by maper long time ago and existence of this suit was forgotten.** 

### 2. Explain why current state is bad and why my solution is good.
**Objectively** - code/sprite unused for a long time is bad. Solution to this is either "delete", either "find use" to keep build sleek&clean.
**Subjectively** - some of TG players find this suit good and would like to use it. 
**Objectively** - satisfaction of player needs by changes that doesn't interfere with balance and doesn't interfere with experience of players who don't need it - good.

**Concluding all of the above, I made this PR. Next time when I'll do same harmless PR I'll know that I need to describe the reasons this way. Thanks.**

**Comment by AseFantome** [comment]
There also might be part about "justifications that can be trivially negated" and "changes that take moderately low effort to implement <...> require a disproportionate justification" with examples of some same changes or game-changing changes completely ignoring that, but I want my PR merged, so... I'll not talk about it.

**Comment by AseFantome** [comment]
Once again, mister @optimumtact, what can you say about this all? Are you decision about closing this PR is final?

**Comment by optimumtact** [comment]
yes it was.

**Comment by AseFantome** [comment]
@Mothblocks, @ninjanomnom, hello, I want to appeal.

**Comment by AseFantome** [comment]
Ah, yes, the main point of PR I forgot to point out _(he forgor)_ is to make suit accessible on other maps, not just MetaStation. 
If maintainers find vendor solution wrong (due to "rarity") I think adding one fixed spawn (like on Meta rn) to every map will work just fine.

**Comment by moocowswag** [comment]
An important question that hasnt been asked yet is WHY this article of clothing only exists as a stagnant spawn on a single map, is it by definitive choice to be limited to one map and accessibility or was it simply just an item that a mapper sprited and only knew how to map it in.

**Comment by Chef-SS** [comment]
sisyphus

**Comment by Mothblocks** [comment]
@Chef-SS turn on nearest neighbor when resizing a sprite or it looks bad

**Comment by DrDiasyl** [comment]
Looks like this PR will only get revived in like 3 years.
Well, maybe I will repsrite this suit then and PR the resprite

**Comment by celulamp** [comment]
Just saying that items are tough to craft (cough cough paper frames cost 20 paper and you can't stack paper) or impossible to find without naturally finding them is annoying when you're trying to do a gimmick or make more of the item. Esp when it's as obtuse as THIS ITEM only spawns in maintenance on one map. Would making it a contraband hacked item at least give it a higher chance of being added? I added candy corn as a chem recipe because it was otherwise unobtainable, things being craftable or usable is good and we shouldn't make things that have no REAL reason to be scarce. Apologies for the yapping but I really do think this is harmless and telling players to not make "low effort tiny PRs" is a horrible thing because that's what everyone starts out with, tiny PRs and they get a grasp of the code and move onto bigger and better things. I find it very discouraging but maybe I'm just sensitive.

**Comment by pilotmans** [comment]
This suit should be available round start as a part of a quality of life update since there is no good reason to make it rare and having more drip is always nice.
